config/samples/dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml

```diff
@@ -8,6 +8,7 @@ spec:
   # Create this deployment before Ceph is deployed
   # on EDPM nodes in an HCI scenario.
   servicesOverride:
+    - bootstrap
```

Comment:
thanks!

Reply:
should i add `download-cache` here as well?

Reply:
No, thanks. That doesn't need to be covered in this PR.